v1.1/select.md

```diff
@@ -528,6 +528,25 @@ LIMIT 5;
 +----+------------------+
 ~~~
 
+Alternatively, you can use `FETCH FIRST`.
+
+~~~ sql
+> SELECT id, name 
+FROM accounts 
+FETCH FIRST 5 ROWS ONLY; 
+~~~
+~~~
++----+------------------+
+| id |       name       |
++----+------------------+
+|  1 | Bjorn Fairclough |
+|  2 | Bjorn Fairclough |
+|  3 | Arturo Nevin     |
+|  4 | Arturo Nevin     |
+|  5 | Naseem Joossens  |
++----+------------------+
+~~~
+
 #### Paginate Through Limited Results
```

Comment:
I know this is from before this PR, but I'm not sure we should be advocating this, `OFFSET` is rarely what you want - it's not particularly efficient and it's sort of error prone.

Reply:
Would be glad to improve this––what do you recommend instead?

Reply:
It depends on the use case, really, but the inefficiency is that the database has to process all the offset-ed rows as well - `SELECT ... LIMIT 1 OFFSET 1000000000` will have to process `1000000001` rows. One better way is to store the sort key of the last row processed and then `SELECT ... WHERE key > key ORDER BY key ... LIMIT n`

I think there are some other things that can go wrong too, but they're slipping my mind at the moment...

Reply:
https://www.citusdata.com/blog/2016/03/30/five-ways-to-paginate/

Reply:
Derp. This is already tracked in #1067. Sure we'll get to it eventually.

Reply:
Ugh, that's an old issue. But I'm glad it's tracked, and we will eventually get to it.